index.js

```diff
@@ -100,6 +100,14 @@ class PQueue {
 		return this._pendingCount < this._concurrency;
 	}
 
+	get concurrency() {
+		return this._concurrency;
+	}
+
+	set concurrency(newConcurrency) {
+		this._concurrency = newConcurrency;
```

Comment:
Shouldn't it trigger next elements if available?

Reply:
Indeed, but only when concurrency is being increased